test/Microsoft.AspNet.Mvc.FunctionalTests/WebApiCompatShimBasicTest.cs

```diff
@@ -490,15 +506,24 @@ public class WebApiCompatShimBasicTest
             var client = server.CreateClient();
 
             // Act
-            var response = await client.GetAsync("http://localhost/api/Blog/HttpRequestMessage/" + action);
+            var response = await client.GetAsync("http://localhost/api/Blog/HttpRequestMessage/" + action,
+                                                HttpCompletionOption.ResponseHeadersRead);
 
             // Assert
             Assert.Equal(HttpStatusCode.OK, response.StatusCode);
             Assert.NotNull(response.Content);
             Assert.Equal("application/pdf", response.Content.Headers.ContentType.MediaType);
 
-            var actualBody = await response.Content.ReadAsStringAsync();
-            Assert.Equal(expectedBody, actualBody);
+            // Since content is PushStreamContent at the server, 
+            // the response is expected to be sent in chunked encoding format.
+            Assert.Null(response.Content.Headers.ContentLength);
+            //Assert.Equal(true, response.Headers.TransferEncodingChunked);
```

Comment:
FYI, this is failing probably since this is an in-memory test...but we should see chunked header in IIS hosted scenario...so ideally we want to run this test in IIS host only...thoughts?